docs/content/users/usage/faq.md

```diff
@@ -111,13 +111,34 @@ In project A, use `mysql -h ddev-projectb-db` to access the database server of p
 
 Let’s say we have two projects, for example: project A, and project B.
 
-To enable server-side HTTP/S communication (i.e. server-side API calls) between projects you can:
+DDEV v1.24.9+ has built-in support for inter-project HTTP/S communication:
+
+```bash
+# call from project A web container to project B's web container
+ddev exec curl https://projectb.ddev.site
+
+# or using the internal Docker network hostname
+ddev exec curl https://ddev-projectb-web
+```
+
+!!!tip "How to use HTTPS from a non-web container"
+    You can use HTTP without extra setup, but to use HTTPS, the container you are calling from must trust the `ddev-webserver` certificate authority (CA).
+
+    ```bash
+    $ ddev exec -s some-service curl https://projectb.ddev.site
+    curl: (60) SSL certificate problem: unable to get local issuer certificate
+    More details here: https://curl.se/docs/sslcerts.html
+    ```
+
+    See [Third Party Services May Need To Trust `ddev-webserver`](../extend/custom-compose-files.md#third-party-services-may-need-to-trust-ddev-webserver).
+
+For DDEV v1.24.8 and earlier, to enable server-side HTTP/S communication (i.e. server-side API calls) between projects you can:
```

Comment:
IMO we don't have to include this any more. The earlier section says clearly v1.24.9+